core-blocks/image/style.scss

```diff
@@ -13,4 +14,21 @@
 		margin-right: auto;
 		text-align: center;
 	}
+
+	&.is-resized {
+		width: min-content;
+
+		// Emulate min-content for Edge and IE11
+		display: -ms-inline-grid;
+		-ms-grid-columns: min-content;
+
+		> :nth-child(2) {
```

Comment:
I'm actually not 100% sure, it's part of the edge trick. But I'm wondering if maybe it's  supposed to target the figcaption, in which case that would be better to write. I'll do some testing, see if we can get away with something cleaner.

Reply:
Great catch — this was supposed to target _everything after the first image_, but since there's only ever a figcaption, it's much cleaner to do just that, and the inline-block was unnecessary also.